pkg/telemetry/prometheus.go

```diff
@@ -0,0 +1,62 @@
+package telemetry
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/HeavyHorst/remco/pkg/log"
+	"github.com/armon/go-metrics"
+	metricsPrometheus "github.com/armon/go-metrics/prometheus"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+type PrometheusSink struct {
+	Addr       string
+	Expiration int
+
+	httpServer     *http.Server
+	prometheusSink *metricsPrometheus.PrometheusSink
+}
+
+func (p *PrometheusSink) Init() (metrics.MetricSink, error) {
+	if p == nil {
+		return nil, ErrNilConfig
+	}
+
+	handler := http.NewServeMux()
+	handler.Handle("/metrics", promhttp.Handler())
+	p.httpServer = &http.Server{Addr: p.Addr, Handler: handler}
+
+	go func() {
+		err := p.httpServer.ListenAndServe()
+		if err != nil && err != http.ErrServerClosed {
+			log.Error(fmt.Sprintf("error starting prometheus server: %v", err))
```

Comment:
I think we only start the prometheus stats endpoint and not a real prometheus server.
Could you please clarify this in this log message?